portal/exercise_diet/static/css/styles.css

```diff
@@ -571,8 +571,10 @@ h2 a {
 	border-bottom-right-radius: 3px;
 	border-top-left-radius: 0;
 	border-top-right-radius: 0;
-	
+    -webkit-transition: margin 200ms, padding 200ms;
+    transition: margin 200ms, padding: 200ms;
```

Comment:
@OptimusRhine 
I think there's an extra `:` after `padding` here
([this is what Codacy found](https://www.codacy.com/app/ivanc/true_nth_usa_portal/pullRequest?prid=1178335))

Reply:
@ivan-c  Ivan, yeah, my bad, thank u for finding that.  @OptimusRhine adam, thank u for the fix.